web/packages/core/src/internal/player/inner.tsx

```diff
@@ -858,6 +849,32 @@ export class InnerPlayer {
         }
     }
 
+    // The effective config loaded upon `load`, possibly changing if `window.RufflePlayer` changes later.
```

Comment:
`loadedConfig` needs to be able to change after `load` in the event that `load` is run either before the extension loads or before a self-hosted script that sets `window.RufflePlayer.config` runs.